tortoise/tests/test_transactions.py

```diff
@@ -0,0 +1,23 @@
+import sqlite3
+
+from tortoise.contrib.test import TestCase
+from tortoise.tests.testmodels import Tournament
+
+
+class TestTransactions(TestCase):
+
+    async def test_transactions(self):
+        try:
+            async with self.db.in_transaction() as connection:
+                event = Tournament(name='Test')
+                await event.save(using_db=connection)
+                await Tournament.filter(id=event.id).using_db(
+                    connection).update(name='Updated name')
+                saved_event = await Tournament.filter(
+                    name='Updated name').using_db(connection).first()
+                self.assertEquals(saved_event.id, event.id)
+                await connection.execute_query('SELECT * FROM non_existent_table')
+        except sqlite3.OperationalError:
```

Comment:
This will work only on sqlite runner, need to go with more broad exception, so this test will work with postgres

Reply:
Doh, obviously.
Yes, this seems straightforward to fix.